libbeat/outputs/logstash/client.go

```diff
@@ -133,6 +139,8 @@ func (l *lumberjackClient) publishWindowed(events []common.MapStr) (int, error)
 		}
 	}
 
+	ackedEvents.Add(int64(len(events)))
```

Comment:
better do counting in PublishEvents. this function returns number of events being send and ACKed